libraries/standard/mqtt/cbmc/proofs/MQTT_GetIncomingPacketTypeAndLength/MQTT_GetIncomingPacketTypeAndLength_harness.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Insert copyright notice
+ */
+
+/**
+ * @file MQTT_GetIncomingPacketTypeAndLength_harness.c
+ * @brief Implements the proof harness for MQTT_GetIncomingPacketTypeAndLength function.
+ */
+#include "mqtt.h"
+
+extern int32_t NetworkInterfaceReceiveStub( NetworkContext_t context,
+                                            void * pBuffer,
+                                            size_t bytesToRecv );
+
```

Comment:
I'd create a NetworkInterfaceReceiveStub.h in the cbmc/includes folder.  I'll bet you are going to use this stub a lot of places.

Reply:
It appears I will use it in two more placed, but I will move it to a header nonetheless since it is more than once.